cmd/node/config/economics.toml

```diff
@@ -30,3 +30,4 @@
     MinGasLimit = "50000"
     GasPerDataByte = "1500"
     DataLimitForBaseCalc = "10000"
+    MaxOverPayGasInPercentage = 1.0 #fraction of value 1 - 100%
```

Comment:
We have to standardize all these "percentage" stuff used in the whole project.

Reply:
deleted. float64 is safe and easy.